src/utils.tsx

```diff
@@ -35,7 +35,9 @@ export function convertPropsToMjmlAttributes<P>(
   // mjml uses a different name (css-class) for the same thing.
   const className = (props as any).className;
   if (typeof className === "string") {
-    mjmlProps["css-class"] = joinClassNames(mjmlProps["css-class"], className);
+    mjmlProps["css-class"] = mjmlProps["css-class"]
+      ? joinClassNames(mjmlProps["css-class"], className)
+      : className;
```

Comment:
I think `joinClassNames` should already handle this. Did you run into an issue with it?

Reply:
Yes, there was an issue with the ts compiler. I'll see if there is an alternative fix if the joinClassNames already handles it

Reply:
I think that's fine. It's probably that `joinClassName` doesn't accept undefined. Which now that I think about it, it might be injecting the string `undefined`.